requirements_image.txt

```diff
@@ -1 +1,3 @@
-scikit-image==0.14.2
\ No newline at end of file
+scikit-image==0.14.2
+requests
```

Comment:
`requests` we can leave in the main `requirements.txt`, just `pypng` we want to keep specific to image use cases.

Reply:
Fixed